src/hotspot/share/gc/g1/g1BiasedArray.hpp

```diff
@@ -103,10 +102,15 @@ class G1BiasedMappedArrayBase : public CHeapObj<mtGC> {
 // heap into this array.
 template<class T>
 class G1BiasedMappedArray : public G1BiasedMappedArrayBase {
+
+  T* base() const { return (T*)G1BiasedMappedArrayBase::_base; }
```

Comment:
Looks like `base() `needs to be public for testing as well :(

Reply:
Sorry for all these issues, I just found out that I did not have gtests configured for my build. Should be fixed now.

I moved the test helper methods into the test.